integration/qbft/tests/msg_router.go

```diff
@@ -1,17 +1,18 @@
 package tests
 
 import (
-	spectypes "github.com/bloxapp/ssv-spec/types"
-	protocolvalidator "github.com/bloxapp/ssv/protocol/v2/ssv/validator"
 	"go.uber.org/zap"
+
+	"github.com/bloxapp/ssv/protocol/v2/ssv/queue"
+	protocolvalidator "github.com/bloxapp/ssv/protocol/v2/ssv/validator"
 )
 
 type msgRouter struct {
 	validator *protocolvalidator.Validator
 }
 
-func (m *msgRouter) Route(logger *zap.Logger, message spectypes.SSVMessage) {
-	m.validator.HandleMessage(logger, &message)
+func (m *msgRouter) Route(logger *zap.Logger, message *queue.DecodedSSVMessage) {
+	m.validator.HandleMessage(logger, message)
 }
```

Comment:
I thought we are not even running these tests, also this seems like code duplication? we have router in p2p. maybe I'm missing something.

Reply:
We run integration tests in `.github/workflows/integration-test.yml`. Or did you mean something else?